app/components/track-additional-info/template.hbs

```diff
@@ -1,6 +1,6 @@
 {{#if (or @ensembleName @conductorName)}}
   <ToggleBox @verticalPosition={{@verticalPosition}} @horizontalPosition={{@horizontalPosition}} @open={{true}} @customIconPlacement={{true}} class="secondary-metadata-toggle-box" as |toggle|>
-    <toggle.trigger>
+    <toggle.trigger @class=" gtm__click-tracking-expand-track-info">
```

Comment:
is the space in front of this intentional?

Reply:
good catch